patientui/src/main/java/fr/marc/patientui/controller/PatientController.java

```diff
@@ -0,0 +1,72 @@
+package fr.marc.patientui.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import fr.marc.patientui.beans.PatientBean;
+import fr.marc.patientui.proxies.PatientInfoProxy;
+
+@Controller
+public class PatientController {
+	
+	private final PatientInfoProxy patientInfoProxy;
+	
+	
+	
+	public PatientController(PatientInfoProxy patientInfoProxy) {
+		super();
+		this.patientInfoProxy = patientInfoProxy;
+	}
+
+
+	@RequestMapping("/")
+	public String home() {
+		return "Home";
+	}
+	
+	@RequestMapping("/PatientList")
+	public String patientListPage(Model model) {
+		List<PatientBean> patients =  patientInfoProxy.getPatients();
+	    model.addAttribute("patients", patients);
+		return "PatientList";
+	}
+	
+	@GetMapping("/PatientSearch")
+	public String patientSearchPage(
+			@RequestParam (name="family", defaultValue="") String family,
+			@RequestParam (name="given", defaultValue="") String given,
+			@ModelAttribute PatientBean patientBean,
+			Model model) {
```

Comment:
nit:

Indentation is almost good, you'd also do it even for the last `) {` .

Ex:
```java
public String patientSearchPage(
			@RequestParam (name="family", defaultValue="") String family,
			@RequestParam (name="given", defaultValue="") String given,
			@ModelAttribute PatientBean patientBean,
			Model model
) {
    // Your code here..
}
```